v1/metrics.yaml

```diff
@@ -2663,6 +2663,228 @@ paths:
 #              agentOptions:
 #                keepAlive: true
 #      x-monitor: false
+  /mediarequests/aggregate/{referer}/{media_type}/{agent_type}/{granularity}/{start}/{end}:
```

Comment:
I wonder if `mediarequests` is the most suitable name here. Since the root of the end point is `/metrics/` perhaps just `media` is enough? Alternatively, other end points under this hierarchy have a hyphen in multi-words, e.g. `edited-pages`, `unique-devices`, etc, so perhaps it would be good to call it `media-requests` for consistency if you think that `requests` is a strictly necessary jargon.

Also, here you have `agent_type`, while in the other end point you call it `agent`. Choose one or the other, as they point to the same type. I personally prefer `agent_type` as it makes it clearer what the parameter is about.